build/msw/wx_setup.props

```diff
@@ -119,5 +121,14 @@
     <BuildMacro Include="wxIntRootDir">
       <Value>$(wxIntRootDir)</Value>
     </BuildMacro>
+    <BuildMacro Include="wxIntRootDir">
+      <Value>$(wxIntRootDir)</Value>
+    </BuildMacro>
```

Comment:
This seems to have been duplicated accidentally?

Reply:
Oops, yes it was...